src/buf.gen.yaml

```diff
@@ -4,7 +4,7 @@ managed:
   go_package_prefix:
     default: github.com/DefangLabs/defang/src/protos
 plugins:
-  - plugin: buf.build/protocolbuffers/go
+  - plugin: buf.build/protocolbuffers/go:v1.36.3
     out: protos
     opt: paths=source_relative
   - plugin: buf.build/bufbuild/connect-go
```

Comment:
what about this one?

Reply:
I didn't think this one was causing the issue, so I left it alone. The latest version is `1.10.0` and it's deprecated, so I don't think it will be changing much in the future.

https://buf.build/bufbuild/connect-go

Reply:
deprecated? are we using the wrong plugin then?

Reply:
We might be. A quick look reveals that this one also exists: https://buf.build/connectrpc/go?version=v1.18.1 But it's far from clear to me if it's a drop-in replacement.